dev/README.md

```diff
@@ -158,18 +162,17 @@ related unit tests in Spark for Java and Python. Any errors will exit with a
 non-zero value. To run, use the following command:
 
 ```shell
-./run_docker_compose.sh spark_integration
-
+docker-compose build conda-cpp
+docker-compose build conda-python
+docker-compose build conda-python-spark
+docker-compose run --rm conda-python-spark
 ```
 
-Alternatively, you can build and run the Docker images seperately. If you
-already are building Spark, these commands will map your local Maven repo
-to the image and save time by not having to download all dependencies. These
-should be run in a directory one level up from your Arrow repository:
+If you already are building Spark, these commands will map your local Maven
+repo to the image and save time by not having to download all dependencies.
 
 ```shell
-docker build -f arrow/dev/spark_integration/Dockerfile -t spark-arrow .
-docker run -v $HOME/.m2:/root/.m2 spark-arrow
+docker-compose run --rm -v $HOME/.m2:/root/.m2 conda-python-spark
```

Comment:
I wouldn't recommend this. Docker will write files with the root user. The local maven cache will be left in a quasi immutable state for the calling user.